menpo/image/base.py

```diff
@@ -2525,17 +2530,38 @@ def pixels_range(self):
 
     def rolled_channels(self):
         r"""
+        Deprecated - please use the equivalent ``as_rolled_channels`` method.
+        """
+        warn('This method is no longer supported and will be removed in a '
+             'future version of Menpo. '
+             'Use .as_rolled_channels() instead.',
+             MenpoDeprecationWarning)
+        return self.as_rolled_channels()
+
+    def as_rolled_channels(self, out_dtype=None):
```

Comment:
The name `.pixels_with_channels_at_back()` seems overly verbose.
What about, instead, making a generic getter  function with a default parameter encouraging the behaviour you want (channels at the front), e.g. `.pixels_to_array(channels_at_end=False)`?